src/clients/api/queries/getXvsVaultWithdrawalRequests/index.ts

```diff
@@ -0,0 +1,19 @@
+import { GetXvsVaultWithdrawalRequestsInput, GetXvsVaultWithdrawalRequestsOutput } from './types';
+import formatToWithdrawalRequest from './formatToWithdrawalRequest';
+
+export * from './types';
+
+const getXvsVaultWithdrawalRequests = async ({
```

Comment:
This doesn't seem to represent withdrawal requests but rather when deposits are available for withdrawal. I think there is a better suited name than WithdrawlRequests